public/js/admin/contactUs.js

```diff
@@ -48,22 +57,38 @@ doneContactUs.forEach((element) => {
       headers: { 'Content-Type': 'application/json' },
     })
       .then((res) => {
-        res.json();
+        if (res.status === 500) {
+          message.textContent = 'There some error , please try again .';
+          popup.classList.add('popup-show');
```

Comment:
@aminalakhsham Here, you could move these two lines into a small helper function:
`showPopup('there was an error')`

Then if showing a pop-up ever gets more complicated (for example, in the future you wanted to also change the text of the "hide" button, or configure it to sometimes show two buttons "ok" and "cancel"), then you have a single place where you can make those enhancements instead of having to make those changes in many places.